addons/microsoft_account/models/microsoft_service.py

```diff
@@ -17,6 +17,19 @@
 DEFAULT_MICROSOFT_AUTH_ENDPOINT = 'https://login.microsoftonline.com/common/oauth2/v2.0/authorize'
 DEFAULT_MICROSOFT_TOKEN_ENDPOINT = 'https://login.microsoftonline.com/common/oauth2/v2.0/token'
 
+RESOURCE_NOT_FOUND_STATUSES = (204, 404)
+
+# Use Postman as proxy to be able to intercept and analyze exchanges with the Outlook API.
+# For that:
+# 1) configure Postman as explained at: https://learning.postman.com/docs/sending-requests/capturing-request-data/capturing-http-requests
+# 2) configure proxies as below
+# 3) add `proxies=DEBUG_PROXIES` to every requests.<METHOD_NAME> calls
+#
+# Note: in case of SSLError, it's possible to also add the argument `verify=False` to all requests calls
+# DEBUG_PROXIES = {
+#     'http': 'http://localhost:5555',
+#     'https': 'https://localhost:5555',
+# }
```

Comment:
Indeed, it would be a good improvement to have a section in the [testing doc](https://www.odoo.com/documentation/15.0/developer/reference/backend/testing.html) explaining how to intercept and analyze Odoo requests (in a global way, not targeted to the Outlook API).

But I suppose it can stay here for the moment, until you (or sbdy else) has the time and motivation to make it (and then we could probably reference our doc here)

Reply:
after discussion with @Feyensv, I will add a chapter in the documentation.
If the chapter is added before merging this PR, I will eventually remove this comment to avoid duplicated info :P